src/line/LineBot.js

```diff
@@ -171,4 +163,33 @@ module.exports = class {
     pickRandom(array) {
         return array && array.length ? array[Math.ceil(Math.random() * array.length) - 1] : null;
     }
+
+    registerMenu(event) {
+        if (_checkAuth(event, "registerMenu")) {
+            let param = this._extractParameter(event.message.text).split(' ');
+            let promise = this.menuManager.add(param[0], param[1]);
+
+            promise.then((message) => {
+                this.bot.replyText(message);
+            }).catch((message) => {
+                this.bot.replyText(message);
+            });
+        } else {
+            this.bot.replyText('권한이 없습니다.');
+        }
+    }
+
+    _checkAuth(event, usage) {
+        let userId = event.source.userId;
+        let userAuthority = "";
+
+        this.auth["whiteList"].some(function (item) {
+            if (item.userId === userId){
+                userAuthority = item.authority;
+                return true;
+            }
+        });
+
+        return this.auth["accessAuthorities"][usage].includes(userAuthority);
```

Comment:
```suggestion
        const AccessAuthorities = "accessAuthorities"
        return this.auth[AccessAuthorities][usage].includes(userAuthority);
```

stringn literal 들 대신에 constant로 선언해서 사용하면 더 좋지 않을까여